runtime/stackmap/fixreturns.c

```diff
@@ -98,6 +98,10 @@ getReturnBytecode(J9ROMClass * romClass, J9ROMMethod * romMethod, UDATA * return
 
 	U_8 * sigData = J9UTF8_DATA(signature);
 	UDATA sigLength = J9UTF8_LENGTH(signature);
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+	U_8 *returnChar = sigData;
+	while (*(returnChar++) != ')')
```

Comment:
I don't understand this line.

Reply:
I've refactored the code using an existing method and a new macro. Let me know if this makes more sense.

Reply:
This looks fine now,